src/__checks__/insert.check.ts

```diff
@@ -20,15 +20,21 @@ const toSnap = <T extends Query<any>>(query: T): ResultSet<T> => {
 };
 
 const foo = defineTable({
-  id: uuid().primaryKey().default(`gen_random_uuid()`),
-  createDate: timestampWithTimeZone().notNull().default(`now()`),
-  name: text().notNull(),
-  value: integer(),
+  columns: {
+    id: uuid().primaryKey().default(`gen_random_uuid()`),
+    createDate: timestampWithTimeZone().notNull().default(`now()`),
+    name: text().notNull(),
+    value: integer(),
+  },
+  defineIndexes: () => ({}),
```

Comment:
These `defineIndexes` are necessary because we check the type of the query later and we need to lock the index type parameter for the table into something so we can put it in the expected type.

Reply:
I don't understand this sentence. Can you clarify?

Reply:
A couple tests in here check the generated `Table` type used in INSERT queries. If we don't specify `defineIndexes()`, it infers one, but the inferred type is kind of crazy (it prints as `{ ...; }`) and I haven't been able to specify a type to match it, so instead I just explicitly passed in the default to force typescript to use the correct default type of `{}`.

This may be moot, though, because this might be fixed by changing the `defineTable` signature in the way you suggested - it might make it easier for Typescript to infer that the default type for indexes when they're missing is `{}`.

Reply:
Yep, turns out we don't need these with the new signature for `defineIndexes()`.